src/index.js

```diff
@@ -39,6 +43,19 @@ class WarmUP {
     }
   }
 
+  archiveDirectory (src, dest, format = 'zip', { archiverOptions = {}, directoryDestPath = false } = {}) {
```

Comment:
Why having some normal properties and then some properties destructure from an object?

I'd flatten it completely `archiveDirectory (src, dest, format = 'zip', archiverOptions = {}, directoryDestPath = false)`